modules/docs/src/Volo.Docs.Admin.Web/Pages/Docs/Admin/docs-notification-hub.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+var hubName = "document-notification-hub";
+
+var connection = new signalR.HubConnectionBuilder()
+    .withUrl("/" + hubName)
+    .withAutomaticReconnect()
+    .configureLogging(signalR.LogLevel.Information)
+    .build();
+
+var docsNotificationManager = $.extend({}, docsNotificationManager, {
+    messageListeners: {
+        notificationProcessMessages: []
+    }
+});
+
+connection.on("receiveNotificationMessage", function (message) {
+    if (!docsNotificationManager.messageListeners.notificationProcessMessages) {
+        return;
+    }
+
+    docsNotificationManager.messageListeners.notificationProcessMessages.forEach(function (fn) {
+        fn({
+            message: message
+        });
+    });
+});
+
+var showNotification = function(notification) {
+    abp.message.success(notification.message);
```

Comment:
I personally think that using SignalR and sending real-time notifications to the UI is lux for this module and it is unnecessary. It also causes a strange user experience. Think that I am on the website (in the document page, or another page with current implementation) and I suddenly see notification messages just because another user re-indexed a document. Notice that with this implementation all users are getting the notification, not only the one has started it.

I think we should drop signalr and real-time notification at all. That's a background job and we don't need to see the result.